SnapchatNavigation/Extensions/UIScrollView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UIScrollView.swift
+//  SnapchatNavigation
+//
+//  Created by Artur Chabera on 05/03/2019.
+//  Copyright © 2019 Artur Chabera. All rights reserved.
+//
+
+import UIKit
+
+extension UIScrollView {
+    static func makeHorizontalFlow(with horizontalControllers: [UIViewController], in parent: UIViewController) -> UIScrollView {
+        let scrollView = UIScrollView()
+        scrollView.backgroundColor = .clear
+        scrollView.isPagingEnabled = true
+        scrollView.bounces = false
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.showsVerticalScrollIndicator = false
+
+        let width = UIScreen.main.bounds.width
+        let height = UIScreen.main.bounds.height
+
+        for controller in horizontalControllers.enumerated() {
+            let xPosition = CGFloat(controller.offset) * width
+            parent.addChild(controller.element)
```

Comment:
Adding VC to scrollView and configuring constraints can be extracted into another method to make logic more clean.